File: Week01/Floor_and_Ceil_Integer_Division/Built_in.cpp
#include<iostream>
#include<cmath>
#include<vector>

using namespace std;


vector<int> divFloorCeil(int a, int b) {
    int floorVal = floor((double) a / b);
    int ceilVal = ceil((double)a / b);

    return {floorVal, ceilVal};
}

int main() {
    int a = -7, b = 2;

    vector<int> res = divFloorCeil(a, b);
    cout << res[0] << " " << res[1] << endl;

    return 0;
}

File: Week01/Floor_and_Ceil_Integer_Division/Built_in.cs
using static System.Console;

class Built_in
{
    public static int[] divFloorCeil(int a, int b)
    {
        int floorVal = (int)Math.Floor((double)a / b);
        int ceilVal = (int)Math.Ceiling((double)a / b);

        return new int[] { floorVal, ceilVal };
    }

    static void Main()
    {
        int a = -7, b = 2;

        int[] res = divFloorCeil(a, b);
        WriteLine(res[0] + " " + res[1]);
    }
}